src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -375,6 +350,7 @@
     <Reference Include="System.ObjectModel" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
Is this used anywhere in the current version of the changes? I am not able to find any references to it.

Reply:
No, good catch. :) Removed.